### PR TITLE
[chore] update merged pull request w/ milestone

### PR DESCRIPTION
**Pull request by codeboten**
Following the conversation in the SIG call on March 22, the github action in this PR will automatically add a merged PR to the first open milestone with `next version` name it finds.

I've tested this code in my fork

### CODE REVIEW

File: .github/workflows/milestone-add-to-pr.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: "open"
+            })
+            for (const milestone of milestones.data) {
+              if (milestone.title == "next version") {
+                await github.rest.issues.update({
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  issue_number: ${{ github.event.pull_request.number }},
```
Comment (evan-bradley) [inline .github/workflows/milestone-add-to-pr.yml:28]: Could you use `context.issue.number` here?

File: .github/workflows/milestone-add-to-pr.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const milestones = await github.rest.issues.listMilestones({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: "open"
+            })
+            for (const milestone of milestones.data) {
+              if (milestone.title == "next version") {
```
[the remark this answers is not in the document]
Comment (codeboten) [reply .github/workflows/milestone-add-to-pr.yml:24]: I'm happy to add a note in here once i make the PR to manage the next release milestone

File: .github/workflows/milestone-add-to-pr.yml (hunk 1 above)
Comment (codeboten) [reply .github/workflows/milestone-add-to-pr.yml:28]: You could, i dont see much difference one way or the other

File: .github/workflows/milestone-add-to-pr.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (evan-bradley) [reply .github/workflows/milestone-add-to-pr.yml:24]: Sounds good!

**Review by evan-bradley — APPROVED** [review]

**Review by Aneurysm9 — APPROVED** [review]

**Pull request merged**